drivers/nvme/nvme.c

```diff
@@ -144,6 +151,7 @@ static EFI_STATUS nvme_drv_init(EFI_SYSTEM_TABLE *st)
 	if (EFI_ERROR(ret))
 		return ret;
 
+	return 0;
```

Comment:
If we return here, there following code will not be executed, right?
If this return is by design, can we use #ifdef to embrace it, then we will not break the original logic.